notification/src/main/kotlin/com/amazon/opendistroforelasticsearch/notification/core/NotificationMessage.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ */
+
+package com.amazon.opendistroforelasticsearch.notification.core
+
+class NotificationMessage(
+  val refTag: String,
```

Comment:
This looks like inconsistent spacing compared to the other `.kt` files?

Reply:
this is as per the checkstyle. extra space would fail checkstyle